Server/ReasnAPI/ReasnAPI/Mappers/EventMapper.cs

```diff
@@ -0,0 +1,54 @@
+﻿using ReasnAPI.Models.Database;
+using ReasnAPI.Models.DTOs;
+
+namespace ReasnAPI.Mappers
+{
+    public static class EventMapper
+    {
+        public static EventDto ToDto(this Event eventToMap)
+        {
+            var tags = eventToMap.Tags
+                .Select(t => new TagDto { Name = t.Name })
+                .ToList();
+
+            var parameters = eventToMap.Parameters
+                .Select(p => new ParameterDto { Key = p.Key, Value = p.Value })

```

Comment:
Same here, maybe use already created funtion `ToDtoList()` in ParameterMapper.

Reply:
changed